lib/filter.js

```diff
@@ -37,6 +45,9 @@ getFilterQuery = function (filter, fields) {
     _.each(filters, function (filterWord) {
       var filterQueryList = [];
       _.each(fields, function (field) {
+        if (settings.autoQuoteSearchterm === true) {
+          filterWord = quoteRegex(filterWord);
+        }
```

Comment:
this needs to be one each up so it doesn't get re-escaped for every field and add \'s to the query. I'm fixing it, renaming and adding the client option :)